lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -71,6 +73,25 @@ bool lldb_private::formatters::swift::SwiftSharedString_SummaryProvider(
       StringPrinter::ReadStringAndDumpToStreamOptions());
 }
 
+using StringSlice = std::pair<uint64_t, uint64_t>;
```

Comment:
Maybe use a `struct { uint64_t start, end; }` here so we know what `first` and `second` represent?

Reply:
Thanks, changed.